inference-engine/src/mkldnn_plugin/nodes/mkldnn_strided_slice_node.cpp

```diff
@@ -157,6 +177,20 @@ void MKLDNNStridedSliceNode::getSupportedDescriptors() {
         fillingInParameters(attrs.end, END_ID, attrs.endDims[0], 0);
     if (attrs.strideDims.size())
         fillingInParameters(attrs.stride, STRIDE_ID, attrs.strideDims[0], 1);
+    if (attrs.axesDims.size()) {
+        fillingInParameters(attrs.axes, AXES_ID, attrs.axesDims[0], 0);
+        std::vector<int> beginTmp(outputRank, 0);
+        std::vector<int> endTmp(outputRank, 0);
+        int i = 0;
+        for (auto a : attrs.axes) {
+            beginTmp[a] = attrs.begin[i++];
```

Comment:
Are you sure that we should increment `i` after initializing `beginTmp[a]` before `endTmp`? Seems like mistake...

Reply:
Sure, fixed.